administrator/components/com_fabrik/views/element/tmpl/bootstrap_control_group.php

```diff
@@ -12,8 +12,28 @@
 // No direct access
 defined('_JEXEC') or die('Restricted access');
 
+$dataShowOn = '';
+if ($this->field->showon)
+{
+	JHtml::_('jquery.framework');
+	JHtml::_('script', 'jui/cms.js', array('version' => 'auto', 'relative' => true));
+    	$dataShowOn = ' data-showon=\'' . json_encode(JFormHelper::parseShowOnConditions($this->field->showon, $this->field->formControl, $this->field->group)) . '\'';
+                    
+	$showOns = JFormHelper::parseShowOnConditions($this->field->showon, $this->field->formControl, $this->field->group);
+
+	if ($this->field->repeat)
+	{
+		foreach ($showOns as &$showOn)
+		{
+			$showOn['field'] .= '[' . $form->repeatCounter . ']';
+		}
+	}
+
+	$dataShowOn = ' data-showon=\'' . json_encode($showOns) . '\'';
```

Comment:
I think this would be better immediately following the statement which sets $showOns.